templates/sabayon.json

```diff
@@ -1,16 +1,15 @@
 {
     "create_message": ["You just created a Sabayon container ",
-                       "(release=RELEASE, arch=ARCH, variant=VARIANT)\n\n",
-                       "To enable sshd, run: rc-update add sshd"],
+                       "(release=RELEASE, arch=ARCH, u=Unprivileged Container)\n\n",
```

Comment:
Yes I know but on lxc-sabayon script I added an option -u because I permit create of unprivileged container directly from lxc script (different of others distro scripts) and with this option I execute some other stuff for systemd to correctly start of sabayon distro on unprivileged container.
See https://github.com/lxc/lxc/blob/master/templates/lxc-sabayon.in#L193

This means that could be correct create two container images for sabayon: one for privileged container and one for unprivileged.

WDYT?